HW03/aleksei_zhuchkov/aleksei_zhuchkov_hw_03_t_03.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Homework3
```

Comment:
You got a very cumbersome solution, maybe my hint will lead you to something

- Split the log into lines
- Select all lines with "Calling"
- iterate through the result array and fetch using a regular TIMESTAMP value
- if the final result is empty then return 0, if not then call the difference calculation method

I think u will write calculate diff method, but if anything, I will check your option and make a review
P.s You can use .each_cons(2)

Reply:
I have done refactor my code.

Reply:
You can see my new code https://github.com/labs-ruby/homeworks-2022/pull/63/commits/b1c90d4f0c7d83b8744bc4973db950cbb69d5dab